### PR TITLE
Cft preview 00 cleanup add to base

### PR DESCRIPTION
**Pull request by reespozzi**
PR to cleanup one cluster configuration and add traefik2 to preview base

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

**Review by timja — APPROVED** [review]
not checked closely

**Review by ciaranca — DISMISSED** [review]

File: apps/admin/preview/base/kustomization.yaml [hunk 1]
```diff
@@ -2,11 +2,21 @@ apiVersion: kustomize.config.k8s.io/v1beta1
...
-  - ../../keda
+  - kube-slack-values.yaml
+  - manage-configmaps-clusterrole.yaml
+
 patchesStrategicMerge:
-  - ../../aat/base/keda-identity.yaml
\ No newline at end of file
+  - ../../aat/base/keda-identity.yaml
+  - ../../traefik2/preview/secret-provider.yaml
+  - ../../traefik2/preview/tlsstore-patch.yaml
```
Comment (thomast1906) [inline apps/admin/preview/base/kustomization.yaml:20]: Is this missing 
`../../aad-pod-id/mi/preview/azure-identity-patch.yaml`

Comment (reespozzi) [reply apps/admin/preview/base/kustomization.yaml:20]: Does it need more adding than `- ../../aad-pod-id/mi` in `/base` @thomast1906 ?

Comment (thomast1906) [reply apps/admin/preview/base/kustomization.yaml:20]: https://github.com/hmcts/cnp-flux-config/tree/master/apps/admin/aad-pod-id/mi/preview ?

This was here previous https://github.com/hmcts/cnp-flux-config/pull/16873/files#diff-75725a845e4c07616bdfbbd8093fb421b149d92aa662a0f14dc677ef9caab870L16 

Can we doublecheck all patch removals etc are included in the base

Comment (reespozzi) [reply apps/admin/preview/base/kustomization.yaml:20]: Added now and base here looks the same as it was in previous patch minus a cluster specific file

**Comment by thomast1906** [comment]
traefikv1 to be removed?
https://github.com/hmcts/cnp-flux-config/tree/cft-preview-00-cleanup-add-to-base/apps/admin/traefik/preview

**Comment by hannah38** [comment]
> traefikv1 to be removed? https://github.com/hmcts/cnp-flux-config/tree/cft-preview-00-cleanup-add-to-base/apps/admin/traefik/preview

Have removed old patches

**Comment by reespozzi** [comment]
cc @thomast1906 re-tagging for review please

**Review by thomast1906 — APPROVED** [review]

**Pull request merged**